ambry-server/src/main/java/com/github/ambry/server/RepairRequestRecord.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2023 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ */
+package com.github.ambry.server;
+
+import java.util.Objects;
+
+
+/**
+ * It is the record of the partially failed request.
+ * It can be a TtlUpdate or a Delete request.
+ * The request was executed successfully on one replica but failed to execute on other replicas.
+ */
+public class RepairRequestRecord {
+  // Blob Id
+  private final String blobId;
+  // Partition id of the blob
+  private final int partitionId;
```

Comment:
let's use long here to keep it consistent.

Reply:
Changed.